src/lib/Gen/adoc/AdocGenerator.cpp

```diff
@@ -69,10 +76,35 @@ build(
     if(! ex)
         return ex.error();
 
-    MultiPageVisitor visitor(*ex, outputPath, corpus);
+    std::string path = files::appendPath(outputPath, "reference.tag.xml");
+    if(auto err = files::createDirectory(outputPath)) 
+    {
+        return err;
+    }
+
+    std::ofstream os;
+    try
+    {
+        os.open(path,
+            std::ios_base::binary |
+                std::ios_base::out |
+                std::ios_base::trunc // | std::ios_base::noreplace
+            );
+    }
+    catch(std::exception const& ex)
+    {
+        return formatError("std::ofstream(\"{}\") threw \"{}\"", path, ex.what());
+    }    
+
+    RawOstream raw_os(os);
+    auto tagfileWriter = TagfileWriter(raw_os, corpus);
```

Comment:
Completely independent behavior. It's not even using the `domCorpus` that includes the reference file names. It shouldn't be made dependent on `build` for "convenience".